recipes/ghc/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include" 
+export LDFLAGS="-L$PREFIX/lib"
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export LIBRARY_PATH="$PREFIX/lib:$LIBRARY_PATH"
```

Comment:
Any reason not to include the existing values for these environment variables?

Reply:
Thanks for reviewing! There is no reason, just added the existing flag values :-)